src/containers/org/CollaborationsParticipationContainer.js

```diff
@@ -0,0 +1,97 @@
+// @flow
+import React, { useState } from 'react';
+
+import debounce from 'lodash/debounce';
+import styled from 'styled-components';
+import { List } from 'immutable';
+import { Colors, SearchInput, Typography } from 'lattice-ui-kit';
+
+import { World2Icon } from '../../assets';
+
+const { NEUTRAL } = Colors;
+
+const BasicListCard = styled.div`
+  align-items: center;
+  background-color: ${NEUTRAL.N50};
+  border-radius: 5px;
+  display: flex;
+  padding: 8px 16px;
+`;
+
+const BasicListCardWrapper = styled.div`
+  margin: 16px 0;
+
+  > div:not(:last-child) {
+    margin-bottom: 8px;
+  }
+`;
+
+const IconWrapper = styled.span`
+  display: inline-flex;
+  margin-right: 16px;
+`;
```

Comment:
You can use `GapGrid` instead (more examples in `OrgContainer`).

```
<GapGrid>
  <World2Icon />
  {title}
</GapGrid>
```

One note - I think `GapGrid` has issues with responsiveness.

Reply:
punting on this until card consolidation